`package.json` is missing

The recommended way to install the package

`npm install --save-dev https://github.com/Fevol/obsidian-typings`

fails

```
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path C:\Users\mnaou\AppData\Local\npm-cache\_cacache\tmp\git-cloneoOzvPX/package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\mnaou\AppData\Local\npm-cache\_cacache\tmp\git-cloneoOzvPX\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

That's because the project is missing `package.json` file

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,1 @@
+node_modules/

--- a/package.json
+++ b/package.json
@@ -17,5 +17,8 @@
   "bugs": {
     "url": "https://github.com/Fevol/obsidian-typings/issues"
   },
-  "homepage": "https://github.com/Fevol/obsidian-typings#readme"
+  "homepage": "https://github.com/Fevol/obsidian-typings#readme",
+  "dependencies": {
+    "obsidian": "^1.4.11"
+  }
 }

--- a/package.json
+++ b/package.json
@@ -1,0 +1,21 @@
+{
+  "name": "obsidian-typings",
+  "version": "1.0.0",
+  "description": "TypeScript typings for undocumented Obsidian API functions",
+  "main": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Fevol/obsidian-typings.git"
+  },
+  "keywords": [
+    "obsidian",
+    "typings",
+    "API"
+  ],
+  "author": "Fevol",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Fevol/obsidian-typings/issues"
+  },
+  "homepage": "https://github.com/Fevol/obsidian-typings#readme"
+}

--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "TypeScript typings for undocumented Obsidian API functions",
   "main": "",
+  "types": "obsidian-ex.d.ts",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Fevol/obsidian-typings.git"

